Tests/ContentKeyCrudTests.cs

```diff
@@ -54,6 +54,12 @@ public void AddContentKey_WithVariousValidData_Succeeds()
 				Value = new byte[Constants.ContentKeyLengthInBytes],
 				ExplicitIv = null
 			})));
+			Assert.Null(Record.Exception(() => document.ContentKeys.Add(new ContentKey
+			{
+				Id = Guid.NewGuid(),
+				Value = new byte[Constants.ContentKeyLengthInBytes],
+				CommonEncryptionScheme = "cenc"
```

Comment:
Perhaps makes sense to check that all other valid schemes pass as well?

Reply:
Now other schemes are checked as well.